src/services/lnd/index.ts

```diff
@@ -102,5 +108,6 @@ export const LndService = (): ILightningService | LightningServiceError => {
     registerInvoice,
     lookupInvoice,
     lookupPayment,
+    deleteFailedPayAttempts,
```

Comment:
Where is this being called? the `LndService` function should only return an object fulfilling [this](https://github.com/GaloyMoney/galoy/blob/main/src/domain/ledger/index.types.d.ts#L87) interface. Which in turn only defines functions that are being used in `src/app` use-cases. IE its part of the re-worked, clean code.
This is why `check-code` is failing.

If you want to add it here you should be wrapping the code with the same `try/catch` + error pattern we have been using in the other cleaned up code sections - but that really depends on where / how its being called.